File: R/tests/testthat/test_omsfree_ewmv.R
context("Test omsfree_ewmv")
library(hBayesDM)

test_that("Test omsfree_ewmv", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(omsfree_ewmv(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_omsfree_ewmv.py
import pytest

from hbayesdm.models import omsfree_ewmv


def test_omsfree_ewmv():
    _ = omsfree_ewmv(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
